src/main/java/com/datalogics/pdf/samples/util/VersionUtils.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2015 Datalogics, Inc.
+ */
+
+package com.datalogics.pdf.samples.util;
+
+import com.adobe.pdfjt.Version;
+
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Utility class that holds methods related to checking the version of PDF Java Toolkit components.
+ */
+public final class VersionUtils {
+
+    private VersionUtils() {}
+
+    /**
+     * Check to see if PDFJT is before version 4.0.0-SNAPSHOT.
+     *
+     * <p>
+     * This is necessary to accommodate both old and new dependencies on PDFJT. Uses the version.properties resource
+     * stored in PDFJT.
+     *
+     * @return is PDFJT before version 4.0.0-SNAPSHOT
+     * @throws IOException an I/O operation failed or was interrupted
+     */
+    public static boolean pdfjtIsBeforeVersion4() throws IOException {
+        try (final InputStream propertiesStream = Version.class.getResourceAsStream("version.properties")) {
```

Comment:
This is practically a duplicate of ``pdfjtIsBeforeVersion``. Please just call that function with ``"4.0.0-SNAPSHOT"`` and eliminate the duplicate code.